internal/adapter/handlebars/handlebars_test.go

```diff
@@ -238,12 +238,13 @@ func TestDateHelper(t *testing.T) {
 	testString(t, "{{date now 'timestamp'}}", context, "200911172034")
 	testString(t, "{{date now 'timestamp-unix'}}", context, "1258490098")
 	testString(t, "{{date now 'cust: %Y-%m'}}", context, "cust: 2009-11")
-	testString(t, "{{date now 'elapsed'}}", context, "13 years ago")
+	testString(t, "{{date now 'elapsed'}}", context, "14 years ago")
 }
 
 func TestGetDateHelper(t *testing.T) {
 	context := map[string]interface{}{"now": time.Date(2009, 11, 17, 20, 34, 58, 651387237, time.UTC)}
-	testString(t, "{{get-date \"2009-11-17T20:34:58\"}}", context, "2009-11-17 20:34:58 +0000 UTC")
+	localOffsetAndTZ := time.Now().Format("-0700 MST")
+	testString(t, "{{get-date \"2009-11-17T20:34:58\"}}", context, "2009-11-17 20:34:58 "+localOffsetAndTZ)
```

Comment:
Test would fail because the string included my local tz and offset, assume this wasn't regularly caught because CI is always in UTC and perhaps you run it locally in a container or whatever. I did turn it into a regexp match in 5574400a0ff7fa77ad0bad8781b3ddd4473ec928 but reverted back to an explicit match as the intention is clearer.

Reply:
Sounds good